drizzlepac/astrodrizzle.help

```diff
@@ -1009,33 +1008,28 @@ ignore pixel flags of 64 and 32 in the DQ array of the (``_flt.fits``)
 images.  Align the final product such that North is
 up, and set the final pixel scale to 0.05 arcseconds/pixel.
 
-1. Run the task from ``PyRAF`` using the ``TEAL`` GUI::
-
-   >>> import drizzlepac
-   >>> epar astrodrizzle
-
-2. Run the task from ``PyRAF`` using the command line. ::
+1. Run the task from python using the command line. ::
 
     >>> import drizzlepac
     >>> from drizzlepac import astrodrizzle
     >>> astrodrizzle.AstroDrizzle('*flt.fits', output='final',
     ...     wcskey='A', driz_sep_bits='64,32', final_wcs=True,
     ...     final_scale=0.05, final_rot=0)
 
-   Or, run the same task from the PyRAF command line, but specify all
+   Or, run the same task from the python command line, but specify all
    parameters in a config file named ``myparam.cfg``:
```

Comment:
I just want to make sure you tested the use of the configuration file in this context.  Please comment.

Reply:
I did not originally, but I did this morning. 

The code runs and attempts to load the custom configuration file using `teal`, but throws an error when run `teal.load()`. 

That being said, i've never been able to run teal on my Mac.